test_data/indent_string_literal_interpolation.good.nix

```diff
@@ -44,16 +44,26 @@
       bla = hoi
     '';
 
-  nested_antiquotation =
-    mkBefore
+  singleAsciiDoc = value: ''
+    ${if lib.hasAttr "example" value then ''
```

Comment:
why does it collapse those lines here?

Reply:
ah thank you for pointing this out. Yeah I think since we ignore most of the formatter inside node string and node interpolation, this shouldn't be formatted like that. Let me fixed it.